ui/src/components/Publication/RelatedPublications/ViewAllModal/index.tsx

```diff
@@ -0,0 +1,219 @@
+                                Most Recent
+                            </span>
+                        </label>
+                        <label htmlFor={relevantInputId} className="flex items-center">
+                            <input
+                                type="radio"
+                                id={relevantInputId}
+                                name={sortOrderInputName}
+                                onChange={() => setSortOrder('relevant')}
+                                defaultChecked={sortOrder === 'relevant'}
+                            />
+                            <span className="text-gray-700 ml-3 block text-sm font-medium sm:text-nowrap">
+                                Most Relevant
+                            </span>
+                        </label>
+                    </div>
+                </fieldset>
+                {user && (
+                    <label htmlFor={ownFilterInputId} className="flex items-center sm:w-1/2 sm:justify-end sm:pt-5">
+                        <input
+                            type="checkbox"
+                            id={ownFilterInputId}
+                            className="rounded-sm"
+                            checked={ownLinks}
+                            onChange={(event) => setOwnLinks(event.target.checked)}
+                        />
+                        <span className="text-gray-700 ml-3 block text-sm font-medium sm:text-nowrap">
+                            View my suggestions
+                        </span>
+                    </label>
+                )}
+            </div>
+            <Framer.AnimatePresence>
+                {getCrosslinksError && (
+                    <Components.Alert
+                        key="getCrosslinksError"
+                        severity="ERROR"
+                        title={getCrosslinksError.response.data.message || getCrosslinksError.message}
+                        className="mb-4"
+                    />
+                )}
+                {genericError && (
+                    <Components.Alert key="genericError" severity="ERROR" title={genericError} className="mb-4" />
+                )}
+                {!getCrosslinksResponse?.data?.length && !isValidating && (
+                    <Components.Alert
+                        severity="INFO"
+                        title="No results found"
+                        details={[
+                            'Try some different search criteria.',
+                            'If you think something is wrong, please contact the helpdesk.'
+                        ]}
+                        className="mb-4"
+                    />
+                )}
+                {getCrosslinksResponse?.data?.length && (
+                    <>
+                        <div className="rounded flex flex-col gap-4">
+                            {getCrosslinksResponse.data.map((crosslink, index: number) => {
+                                let classes = '';
```

Comment:
Hmm, recall leaving a comment about this somewhere else. Using normal if statements rather than ternary operators to update the `classes` variable.

Reply:
That's right, you suggested it on a previous PR. I think this must have been started before that - I should probably merge main in before I create the pull request next time. Anyway, I've done some merging and the if statements are in place here now.